include/comments.php

```diff
@@ -3,12 +3,37 @@
 function createComment($content){
     dbQuery(
         '
-        INSERT INTO comments (content)
-        VALUES(:content)
+        INSERT INTO comments (content, blogPostId)
+        VALUES(:content, :blogPostId)
         ',
         [
-            'content'=>$_REQUEST['content']
+            'content'=>$content,
+            'blogPostId'=>$_REQUEST['blogPostId']
```

Comment:
This should be taken in as a parameter to the function. Functions generally should be looking in the request like this.